lib/fixtures/keycloak.go

```diff
@@ -305,6 +306,21 @@ func SetupKeycloak(ctx context.Context, kcConnectParams KeycloakConnectParams) e
 		return err
 	}
 
+	// Create OpenTDF Public Client
+	_, err = createClient(ctx, client, token, &kcConnectParams, gocloak.Client{
+		ClientID:        gocloak.StringP(opentdfPublicClientID),
+		Enabled:         gocloak.BoolP(true),
+		Name:            gocloak.StringP(opentdfPublicClientID),
+		PublicClient:    gocloak.BoolP(true),
+		ProtocolMappers: &protocolMappers,
+		// TODO: take via CLI arguments?
+		// Note: otdfctl CLI auth code flow runs from localhost:9000)
+		RedirectURIs: &[]string{"http://localhost:9000/*"},
+	}, nil, nil)
+	if err != nil {
+		return err
+	}
```

Comment:
If we're hardcoding the RedirectURIs then I think this should be a `otdfctlPublicClient` like `enable_otdfctl_client` or something like that.

Reply:
I think this public client may also be used by varied PEPs and is not guaranteed to be `otdfctl` only. Note this block of code is only used in the `go build -o platform && ./platform provision keycloak` platform binary command flow (which maybe we should just deprecate rather than continuing to support both?). The `./platform provision keycloak-from-config` redirect_uris are in YAML and not hardcoded: https://github.com/opentdf/platform/pull/1365/files#diff-2d149f4c96fefaec806f5f07b2f6b804903ac1b5f0b90a54cfebe53c088119c0R74

Reply:
TODO: Make `provision keycloak` the primary command that reads from the YAML and deprecate `keycloak-from-config`.